### PR TITLE
DPL: workaround FairMQ Socket::Receive API issue for messages > 2GB

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

**Comment by ktf** [comment]
@shahor02

**Comment by davidrohr** [comment]
actually, this is not save. You have a 2/2^32 chance to hit the error by chance for rare message sizes :) Should be fixed in FairMQ.

File: Framework/Core/src/DataProcessingDevice.cxx [hunk 1]
```diff
@@ -526,7 +526,7 @@ void DataProcessingDevice::doPrepare(DataProcessorContext& context)
         while (true) {
           FairMQParts parts;
           result = fairMQChannel.Receive(parts, 0);
```
Comment (rbx) [inline Framework/Core/src/DataProcessingDevice.cxx:528]: This already returns int64_t: https://github.com/FairRootGroup/FairMQ/blob/v1.4.26/fairmq/FairMQChannel.h#L309

Comment (ktf) [reply Framework/Core/src/DataProcessingDevice.cxx:528]: Agreed. The problem was an int on DPL side.

**Pull request closed without merging**